UPGRADE-1.13.md

```diff
@@ -54,3 +55,15 @@
 
 1. Class `\Sylius\Bundle\ShopBundle\Calculator\OrderItemsSubtotalCalculator` has been deprecated. Order items subtotal calculation
    is now available on the Order model `\Sylius\Component\Core\Model\Order::getItemsSubtotal`.
+
+1. Using Guzzle 6 has been deprecated in favor of Symfony HTTP Client. Therefore, passing Guzzle 6 `GuzzleHttp\ClientInterface`
+   or any other HTTP Client that does not implement `Psr\Http\Client\ClientInterface`
+   to the `Sylius\Bundle\AdminBundle\Controller\NotificationController` as a first argument is deprecated.
+
+1. Not passing `Psr\Http\Message\RequestFactoryInterface` as a second argument
+   of the `Sylius\Bundle\AdminBundle\Controller\NotificationController` is deprecated.
+
+1. Not passing `Psr\Http\Message\StreamFactoryInterface` as a fifth argument
+   of the `Sylius\Bundle\AdminBundle\Controller\NotificationController` is deprecated.
```

Comment:
These three points could be combined under a single one as they relate to one controller